lte/gateway/docker/Dockerfile.mme

```diff
@@ -0,0 +1,340 @@
+################################################################
```

Comment:
Add this file as lte/gateway/docker/mme/Dockerfile to match our general convention in the codebase please

Reply:
@xjtian moved to correct folder